step-templates/pingdom-pause-resume-check.json

```diff
@@ -0,0 +1,96 @@
 \"Pingdom.Username\" -Required -FailOnValidate)\n    $pingdomPassword = [string] (Get-Parameter -Name \"Pingdom.Password\" -Required -FailOnValidate)\n    $pingdomAppKey = [string] (Get-Parameter -Name \"Pingdom.AppKey\" -Required -FailOnValidate)\n\n    $pingdomCheckId = [string] (Get-Parameter -Name \"Pingdom.CheckId\" -FailOnValidate)\n    $pingdomCheckName = [string] (Get-Parameter -Name \"Pingdom.CheckName\" -FailOnValidate)\n    $pingdomCheckPaused = [string] (Get-Parameter -Name \"Pingdom.CheckPaused\" -Default \"False\" -FailOnValidate)\n\n    $apiVersion = \"2.0\"\n    $url = \"https://api.pingdom.com/api/{0}/checks\" -f $apiVersion\n    $securePassword = ConvertTo-SecureString $pingdomPassword -AsPlainText -Force\n    $credential = New-Object System.Management.Automation.PSCredential($pingdomUsername, $securePassword)\n    $headers = @{ \n        \"App-Key\" = $pingdomAppKey\n    }\n\n    If ([string]::IsNullOrEmpty($pingdomCheckId) -and [string]::IsNullOrEmpty($pingdomCheckName)) {\n        Write-Error \"Please specify CheckId or CheckName!\"\n        Exit\n    } \n    \n    # Find check id by name\n\n    If (-Not [string]::IsNullOrEmpty($pingdomCheckName)) {\n        Write-Host \"Getting uptime check list to find check by name: $url\"\n        Try {\n            $response = Invoke-RestMethod -Uri $url -Method Get -ContentType \"application/json\" -Credential $credential -Headers $headers\n        } Catch {\n            Write-Host \"Error occured when getting uptime check list in Pingdom: \" $_\n        }\n\n        $checkFiltered = $response.checks | Where-Object { $_.name -eq $pingdomCheckName }\n        If ($checkFiltered -eq $null) {\n            Write-Warning \"Check with name $pingdomCheckName not found!\"\n            Exit\n        }\n\n        $pingdomCheckId = $checkFiltered.id\n    }\n\n    If ([string]::IsNullOrEmpty($pingdomCheckId)) {\n        Write-Warning \"Check with name $pingdomCheckName was not found!\"\n        Exit\n    }\n\n    # Pause or resume check\n\n    $url += \"/$pingdomCheckId\"\n\n    $apiParameters = @{}\n    If ($pingdomCheckPaused -eq \"True\") {\n        $apiParameters.Add(\"paused\", \"true\")\n    } Else {\n        $apiParameters.Add(\"paused\", \"false\")\n    }\n\n    If ($apiParameters.Count -gt 0) {\n        $queryString = \"\"\n        $apiParameters.Keys | ForEach-Object { \n            $queryString += ($_ + \"=\" + [Web.HttpUtility]::HtmlEncode($apiParameters.Item($_)) + \"&\")  \n        }\n        $queryString = $queryString.Substring(0, $queryString.Length - 1)\n        $url += \"?$queryString\"\n    }\n\n    Write-Host \"Modifying uptime check: $url\"\n    Try {\n        $response = Invoke-RestMethod -Uri $url -Method Put -ContentType \"application/json\" -Credential $credential -Headers $headers\n        Write-Host $response.message\n    } Catch {\n        Write-Error \"Error occured when adding uptime check in Pingdom: \" $_\n    }\n\n    Write-Host \"End PingdomPauseResumeCheck\"\n}",
```

Comment:
The function `Test-Any` isn't called anywhere (that I can see), can you please remove it